modules/javafx.controls/src/test/java/test/javafx/scene/control/ScrollBarTest.java

```diff
@@ -415,6 +415,34 @@
         assertEquals(scrollBar.getValue(), 50.0, 0.0);
     }
 
+    /**
+     * @test
+     * @bug 8090158
+     * Adjusting the scrollbar value close to the max value should use the block increment and reach the max value.
+     */
+    @Test
+    public void incrementCloseToMax() {
+        scrollBar.setMin(0.0);
+        scrollBar.setMax(100.0);
+        scrollBar.setValue(90.0);
+        scrollBar.adjustValue(0.95); //This should block increment to the max value
+        assertEquals(scrollBar.getValue(), 100.0, 0.0);
```

Comment:
Minor : First two parameters to assertEquals are - expected value and actual value respectively.
You should reorder scrollBar.getValue() and 100.0.